packages/apollo-engine-reporting/src/extension.ts

```diff
@@ -225,15 +228,22 @@ export class EngineReportingExtension<TContext = any>
             node = specificNode;
           }
         }
-        node!.error!.push(
-          new Trace.Error({
-            message: error.message,
-            location: (error.locations || []).map(
-              ({ line, column }) => new Trace.Location({ line, column }),
-            ),
-            json: JSON.stringify(error),
-          }),
-        );
+
+        // With noErrorTraces, the Engine proxy strips all error information
```

Comment:
I wouldn't explain the behavior by comparing it to the `noErrorTraces` option in the Engine proxy, we should just describe what it does.

Reply:
Okay that makes sense. I'll include the comment in the commit message. I think it's important to understand the context around how AS2 differs from the proxy.